tests/shouldwork/Issues/T2593.hs

```diff
@@ -0,0 +1,14 @@
+module T2593 where
+
+import Clash.Prelude
+import Clash.Annotations.SynthesisAttributes
+
+topEntity ::
+  Signal System Bit ->
+  Signal System Bit `Annotate` 'StringAttr "breaka" "me"
```

Comment:
```suggestion
  Signal System Bit `Annotate` 'StringAttr "break" "me"
```
:-)

Reply:
It's a-me!

Reply:
`break` is a systemverilog keyword, we cannot use it as a property name.

Reply:
> It's a-me!

<img src="https://github.com/clash-lang/clash-compiler/assets/3692925/362281bc-8ff6-4638-91e4-baa7044c9204" width="83" height="92">